src/pytorch_lightning/loggers/wandb.py

```diff
@@ -32,10 +32,11 @@
 
 try:
     import wandb
+    from wandb.sdk.lib import RunDisabled
```

Comment:
Yes, we decided to have no dependency on the third party libraries for loggers in tests. This simplifies testing greatly and makes it easier for people to run the tests or add new ones. On top of that, all we provide is wrappers of these apis so there is no need for us call the real objects in most cases. Finally, many loggers connect to live services which means we would have to introduce a lot of mocking anyway.

Reply:
Good catch! We would need to bump the mininum version to `wandb==0.10.19` to allow the import of `RunDisabled`.

Reply:
@awaelchli, @carmocca, @Borda are we fine with bumping to `wandb>=0.10.19`? It's from Feb 2021, so about year and a half old

Reply:
fine with me

Reply:
Yes, we should probably bump directly to 0.10.22 and then remove `_WANDB_GREATER_EQUAL_0_10_22` and associated logic in a separate PR.

Reply:
I've just bumped `wandb` to `0.10.22`, and I can do likely do the other PR getting rid of `_WANDB_GREATER_EQUAL_0_10_22` when this is merged.

Note I also realised that when using python >= 3.9 then `wandb` should be `>=0.10.30` because of an [ invalid import](https://github.com/wandb/client/blob/05d80c7027f26d6f5c81f241d5b8c92889752027/wandb/vendor/watchdog/utils/bricks.py#L175) of `collections` which was moved to `collections.abc` in python 3.9+.

Reply:
sounds good, we can address this separately.